src/main/resources/db/migration/v0.10.0.1__agr_curation_api.sql

```diff
@@ -0,0 +1,9 @@
+DELETE FROM diseaseannotation_conditionrelation;
+
+DELETE FROM conditionrelation_experimentalcondition;
+
+DELETE FROM experimentalcondition;
+
+DELETE FROM conditionrelation;
+		
+		
```

Comment:
I believe this migration would effectively strip disease annotations that have conditionrelations from their conditionrelations, and leave the disease annotation? Would a disease annotation reload afterwards then repopulate those diseaseannotation-conditionrelation links, or would this migration need to be edited to remove the disease annotations with conditionrelations? Also, would there be a less brute-force migration possible (like setting the values of only the experimentalcondition uniqueId to null rather than removing all experimental condition records)?

Reply:
Yes, that's exactly what this migration is intended to do.  DA reload will reattach all condition relations and experimental conditions.   A less brute-force method would not achieve the cleanup of duplicate experimental conditions required.